pages/design-tokens/typesetting/font-family.md

```diff
@@ -155,11 +155,11 @@ Role-based tokens set the font family value based on the _role_ the face plays i
   </table>
 </div>
 
-{:.bg-gold-20v.padding-2.radius-md}
-**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. This documentation is coming soon. See the inline documentation in `_uswds-theme-typography` for more details.
+{:.site-note.margin-top-3}
+**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. Create a new project-specific theme settings file. If your project already has a single project-specific theme settings file, you’re all set. If not, create a new file in your project theme file called `_uswds-theme.scss`.
```

Comment:
Note: This section was updated in #1905. We should make sure to merge in `main`. The `settings-typography` reference there should be ok to leave because it has helpful instructions in the comments.